tests/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+# This is the cmake build file for the tests directory of the
+# NCEPLIBS-g2c_UTILS project.
+#
+# Ed Hartnett, 7/11/21
+
+add_executable(tst_g2_create tst_g2_create.c)
```

Looking at it now, instead of `target_inlcude_directories`, one should link with the g2c library compiled in the project.

```suggestion
add_executable(tst_g2_create tst_g2_create.c)
target_link_libraries(tst_g2_create PRIVATE g2c::g2c)
```

If you were to use `target_include_directories`, this is the correct call:
```
target_include_directories(tst_g2_create PRIVATE $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/src>)
```

---

This builds and passes.